Alignment/MillePedeAlignmentAlgorithm/interface/MillePedeFileReader.h

```diff
@@ -169,7 +169,7 @@ class MillePedeFileReader {
 
 const std::array<std::string, 8> coord_str = {{"X", "Y", "Z", "theta_X", "theta_Y", "theta_Z", "extra_DOF", "none"}};
 inline std::ostream& operator<<(std::ostream& os, const AlignPCLThresholds::coordType& c) {
-  if (c >= AlignPCLThresholds::endOfTypes || c < 0)
+  if (c >= AlignPCLThresholds::endOfTypes || c < AlignPCLThresholds::X)
```

Comment:
we can drop `|| c<0` part as it is always `false` 
```
Alignment/MillePedeAlignmentAlgorithm/interface/MillePedeFileReader.h:172:48: error: result of comparison of unsigned enum expression < 0 is always false [-Werror,-Wtautological-unsigned-enum-zero-compare]
```

Reply:
The whole `if` statement seems to be redundant (unless something casts an integer into the enumeration without bounds checking)
https://github.com/cms-sw/cmssw/blob/dcdf62c803c6e74bdebb31cfbae0021aeaaa5a57/CondFormats/PCLConfig/interface/AlignPCLThresholds.h#L14